src/main/java/com/commercetools/paymenttoorderprocessor/dto/PaymentTransactionCreatedOrUpdatedMessage.java

```diff
@@ -0,0 +1,67 @@
+package com.commercetools.paymenttoorderprocessor.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.messages.GenericMessageImpl;
+import io.sphere.sdk.messages.MessageDerivateHint;
+import io.sphere.sdk.messages.UserProvidedIdentifiers;
+import io.sphere.sdk.payments.Payment;
+import io.sphere.sdk.payments.Transaction;
+import io.sphere.sdk.payments.TransactionState;
+
+import java.time.ZonedDateTime;
+
+@JsonDeserialize(
+        as = PaymentTransactionCreatedOrUpdatedMessage.class
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class PaymentTransactionCreatedOrUpdatedMessage extends GenericMessageImpl<Payment> {
+    public static final String MESSAGE_TYPE = "PaymentTransactionCreatedOrUpdatedMessage";
```

Comment:
These 2 parameters MESSAGE_TYPE, MESSAGE_TYPE does not seem used, is it a reflection mechanism.

Reply:
https://github.com/commercetools/commercetools-payment-to-order-processor/pull/71/commits/f38175a805c909caef0246ee9efe9164f0b1adbb